contracts/GenArt721CoreV3.sol

```diff
@@ -871,7 +877,8 @@ contract GenArt721CoreV3 is
     {
         Project storage project = projects[_projectId];
         require(_scriptId < project.scriptCount, "scriptId out of range");
-        project.scripts[_scriptId] = _script;
+        // store script in contract bytecode
+        project.scriptBytecodeAddresses[_scriptId] = _script.writeToBytecode();
```

Comment:
Making direct note of this for posterity as follow up from our call, this is one call-site where we'd want to `selfdestruct` the old script/script-chunk that is being replaced.

Reply:
This is addressed now with folding in of https://github.com/ArtBlocks/artblocks-contracts/pull/306 into this PR.